### ISSUE TITLE
where is remote_key

### ISSUE THREAD

**Issue by mengds2016** [maximkulkin/esp-ir-tv#1]
I have a problem.I only see power-key and four hdmi-key in the iphone home.where is other remote-key.need apple TV remote?

**Comment by maximkulkin** [comment]
Yes, the other stuff is available through Apple Remote app, although it's not intuitive to find (I guess, it's intuitive if you use remote with Apple TV, remote app looks close to physical Apple TV remote). Volume control though is done by physical volume buttons, no on-screen UI is displayed. I haven't found how to mute, physical mute switch does not seem to mute stuff.

![IMG_0452](https://user-images.githubusercontent.com/10785/68643213-95695c80-04c5-11ea-9a38-4bbef6b344e2.png)
![IMG_0450](https://user-images.githubusercontent.com/10785/68643229-a0bc8800-04c5-11ea-9170-873d1d26fabb.png)

**Comment by mengds2016** [comment]
Thank you
